chaco/tests/test_image_plot.py

```diff
@@ -108,48 +113,56 @@ def plot_comparison(input_image, expected_image, **plot_kwargs):
     plt.show()
 
 
+@unittest.skipIf(six.PY3, "Bug in the image plotter in python 3")
```

Comment:
That's actually a pretty old [issue](https://github.com/enthought/enable/issues/283)